libs/tracker/llmstudio_tracker/session/models.py

```diff
@@ -1,15 +1,35 @@
+from datetime import datetime, timezone
+
+from llmstudio_tracker.config import DB_TYPE
 from llmstudio_tracker.database import Base
+from llmstudio_tracker.db_utils import JSONEncodedDict
 from sqlalchemy import JSON, Column, DateTime, Integer, String
-from sqlalchemy.sql import func
 
 
 class SessionDefault(Base):
     __tablename__ = "sessions"
-    message_id = Column(Integer, primary_key=True, index=True)
-    session_id = Column(String, index=True)
-    chat_history = Column(JSON)
-    extras = Column(JSON)
+
+    if DB_TYPE == "bigquery":
+        message_id = Column(
+            Integer,
+            primary_key=True,
+            default=lambda: int(datetime.now(timezone.utc).strftime("%Y%m%d%H%M%S%f")),
+        )
+        session_id = Column(String)
+        chat_history = Column(JSONEncodedDict)
+        extras = Column(JSONEncodedDict)
+
+    else:
+        message_id = Column(Integer, primary_key=True, index=True)
+        session_id = Column(String, index=True)
+        chat_history = Column(JSON)
+        extras = Column(JSON)
+
     updated_at = Column(
-        DateTime(timezone=True), onupdate=func.now(), server_default=func.now()
+        DateTime(timezone=True),
+        onupdate=lambda: datetime.now(timezone.utc),
```

Comment:
make sure it works for python3.12 and python3.11. I had an issue with datetime on 3.11